pkg/cmd/server/origin/master.go

```diff
@@ -776,6 +776,7 @@ func (c *MasterConfig) RunDNSServer() {
 		glog.Fatalf("Could not start DNS: %v", err)
 	}
 	config.DnsAddr = c.Options.DNSConfig.BindAddress
+	config.NoRec = true // do not want to deploy an open resolver
```

Comment:
I put this in NewServerDefaults in pkg/dns/server.go... seemed to make more sense to me there, but I'm not sure what the distinction between the 2 is supposed to be.